chapter_preliminaries/probability.md

```diff
@@ -1,304 +1,751 @@
+to a subset of the underlying sample space.
+Thus the occurrence where the random variable $X$
+takes value $v$, denoted by $X=v$, is an *event*
+and $P(X=v)$ denotes its probability.
+Sometimes this notation can get clunky,
+and we can abuse notation when the context is clear.
+For example, we might use *P(X)* to refer broadly
+to the *distribution* of $X$, i.e., 
+the function that tells us the probability
+that $X$ takes any given value. 
+Other times we write expressions 
+like $P(X,Y) = P(X) P(Y)$,
+as a shorthand to express a statement 
+that is true for all of the values
+that the random variables $X$ and $Y$ can take, i.e.,
+for all $i,j$ $P(X=i \textrm{ and } Y=j) = P(X=i)P(Y=j)$.
+Other times, we abuse notation by writing
+$P(v)$ when the random variable is clear from the context. 
 Since an event in probability theory is a set of outcomes from the sample space,
 we can specify a range of values for a random variable to take.
 For example, $P(1 \leq X \leq 3)$ denotes the probability of the event $\{1 \leq X \leq 3\}$.
-By definition it is also the probability that the random variable $X$ can take a value from $\{1, 2, 3\}$.
-
-Note that there is a subtle difference between *discrete* random variables, like the sides of a die, and *continuous* ones, like the weight and the height of a person. There is little point in asking whether two people have *exactly* the same height. If we take precise enough measurements you will find that no two people on the planet have the exact same height. In fact, if we take a fine enough measurement, you will not have the same height when you wake up and when you go to sleep. So there is no point in asking about the probability
-that someone is 1.801392782910287192 meters tall. Given the world population of 8 billion humans the probability is virtually 0. It makes more sense in this case to ask whether someone's height falls into a given interval, say between 1.79 and 1.81 meters. In these cases we quantify the likelihood that we see a value as a *density*. The height of exactly 1.80 meters has no probability, but nonzero density. In fact, the probability is then given by the *integral* over the density in the interval of $[1.79, 1.81]$ meters. 
-For more details on continuous random variables see :numref:`sec_random_variables`. 
 
-## Multiple Random Variables
 
-Frequently we will want to consider more than one random variable at a time.
-For instance, we may want to model the relationship between diseases and symptoms. Given diseases, say (flu, COVID-19) and symptoms, say (cough, sneezing, fever), combinations of the two such as (flu, fever) may or may not occur in a patient. Our goal might be to estimate their relationships for better medical care. 
+Note that there is a subtle difference 
+between *discrete*-valued random variables,
+like flips of a coin or tosses of a die, 
+and *continuous*-valued ones,
+like the weight and the height of a person
```

Comment:
*discrete*-valued -> *discrete*
*continuous*-valued -> *continuous*
See both terms in 3.1 and 3.2 in https://en.wikipedia.org/wiki/Random_variable

Reply:
seems excessive and clunky. better to just hyperlink the words "discrete" and "continuous" but without the extra baggage

Reply:
sure that's fine